Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -358,7 +358,20 @@ public class NavigatorIndex {
             default: self = .article
             }
         }
-        
+
+        /// Whether this page kind references a symbol.
+        var isSymbolKind: Bool {
+            // This switch is written with the non-symbol kinds written explicitly on the assumption
+            // that new symbol kinds will be added over time, but that non-symbol kinds are much
+            // more rare to add.
+            switch self {
```

Comment:
Would it be more future-proof if we spelled out all the cases here and didn't use a `default` case? The problem with this approach, while more concise for today, is that in the future if we add a new non-symbol page type we might forget to read this switch statement and add it here also. But if we list all the cases here explicitly the compiler will remind our future-selves to look at this again.

Reply:
I like that suggestion; that way we don't have to rely on people to Just Know that this exists. I'll make the update.